moveit_core/controller_manager/include/moveit/controller_manager/controller_manager.h

```diff
@@ -183,7 +183,7 @@ class MoveItControllerManager
   {
   }
 
-  virtual void initialize(std::shared_ptr<rclcpp::Node>& node) = 0;
+  virtual void initialize(const rclcpp::Node::SharedPtr& node) = 0;
```

Comment:
I'll separate this into a stand-alone PR, once I finish the ros_control package

Reply:
I don't think it's necessary to separate this